COM-3714 - changing the API call to ForStorepick as we need to list all the location that have StorePick option

kibo-DeepakTajane opened the pull request:
As quotes should not depend on the Inventory, All Store Pick location should be listed.
Changed the API to ForStorePick, which will list all Store Pick location that are not disabled.

sagarraut0007 commented on scripts/modules/b2b-account/edit-quote.js:
```diff
@@ -1025,34 +1015,36 @@ define([
                             "address": fulfillmentInfo.fulfillmentContact.address
                         }
                     }));
+            }
```
Can you remove all the unnecessary changes from your PR. looks like some changes happened due to formatting but remove those changes? Only keep the changes which you've made.

kibo-DeepakTajane replied:
remove the formatting changes

sagarraut0007 commented on templates/modules/b2b-account/quotes/edit-quote/product-picker-table.hypr.live:
```diff
@@ -103,7 +103,10 @@
         {% if item.fulfillmentMethod != "Digital" %}
           <select class="mz-l-formfieldgroup-halfsize" data-mz-value="fulfillmentMethod" data-mz-quote-item="{{ item.id }}" >
             <option {% if item.fulfillmentMethod=="Ship" %} selected {% endif %} value="Ship">Direct Ship</option>
```
Can you also take the labels "Direct Ship" and "Pickup" from localizations JSON files?

kibo-DeepakTajane replied:
moved them to label file

kibo-DeepakTajane committed:
COM-3714 - Empty 'Store Locator' window on edit quote with no inventory

As quotes should not depend on the Inventory, All Store Pick location should be listed.
Changed the API to ForStorePick, which will list all Store Pick location that are not disabled.

sagarraut0007 commented on labels/en-US.json:
```diff
@@ -340,6 +340,8 @@
   "friday": "Friday",
   "from": "from",
   "fromDeliveryLocation": "from location",
+	"fulfillmentDirectShip": "Direct Ship",
```
fix indentation over here. And you also have to add this to another localization file as well apart from english.

kibo-DeepakTajane replied:
done

sagarraut0007 replied:
@kibo-DeepakTajane , Based on previous comment, you've missed adding these new labels into "de-DE.json". see my original comment.

kibo-raghu reviewed (approved):
Changes Approved
